src/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import './TodoList.scss';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <ul className="todo-lists">
+    {todos.map(todo => (
+      <li key={todo.id} className="todo-list">
```

Comment:
it's one todo, not list
```suggestion
      <li key={todo.id} className="todo">
```